src/librustdoc/html/static/js/main.js

```diff
@@ -279,7 +279,8 @@ function preLoadCss(cssUrl) {
             const params = {};
             window.location.search.substring(1).split("&").
                 map(s => {
-                    const pair = s.split("=");
+                    // https://github.com/rust-lang/rust/issues/119219
+                    const pair = s.split("=").map(x => x.replace(/\+/g, " "));
```

Comment:
Thanks to this, I discovered that it was okay to use `+` to replace a space in a query parameter but not in the path. So weird.